File: CodingPractice-master/leetcode/algorithms/cpp/0268_MissingNumber.c
/*
* @Author: Yan_Daojiang
* @Date:   2019-05-17 09:57:02
* @Last Modified by:   13738
* @Last Modified time: 2019-05-17 09:57:13
*/
int missingNumber(int* nums, int numsSize){
    int s=0,f=0;
    for(int i=0;i<=numsSize;i++)
        s+=i;
    for(int i=0;i<numsSize;i++)
        f+=nums[i];
    return s-f;
}

File: CodingPractice-master/leetcode/algorithms/python/0268_MissingNumber.py
# -*- coding: utf-8 -*-
# @Author: Yan_Daojiang
# @Date:   2019-05-17 09:57:44
# @Last Modified by:   Yan_Daojiang
# @Last Modified time: 2019-05-17 09:58:05
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        s=0
        for i in range(len(nums)+1):
            s=s+i
        return s-sum(nums)